htdocs/fourn/class/fournisseur.facture.class.php

```diff
@@ -238,14 +252,29 @@ class FactureFournisseur extends CommonInvoice
 	 */
 	public $fournisseur;
 
+	// Multicurrency
+	/**
+	 * @var int ID
+	 */
+	public $fk_multicurrency;
+
+	public $multicurrency_code;
```

Comment:
Properties for multicurrency should be defined into CommonInvoice class

Reply:
correction done, I hve also put the inherit of four invoice line on comon invoice lines instead of commonobjectlines. 
I have resolved all the overide on the commoninvoice class.